addon/authenticators/oidc.js

```diff
@@ -50,17 +50,19 @@ export default BaseAuthenticator.extend({
       );
     }
 
-    const data = await this.get("ajax").post(getUrl(tokenEndpoint), {
-      responseType: "application/json",
-      contentType: "application/x-www-form-urlencoded",
-      data: {
-        code,
-        client_id: clientId,
-        grant_type: "authorization_code",
-        redirect_uri: this.redirectUri
-      }
+    const body = new URLSearchParams();
```

Comment:
Sadly, this will not work in IE11 since `URLSearchParams` [is not supported there](https://caniuse.com/#feat=urlsearchparams).. We now have two options for fixing this:

1. Polyfill `URLSearchParams`
2. Use a different approach for creating the POST body. E.g using `encodeURIComponent`, `reduce` and `join`

I'd prefer the second approach since we don't really want to enforce someone to polyfill so we'd have to make it configurable. Second approach is easier to maintain and implement.

Reply:
Agreed, I will refactor to use `encodeURIComponent`.